### PR TITLE
Webui integration

### PR DESCRIPTION
**Pull request by chrismattmann**

### CODE REVIEW

File: ivy/ivy.xml [hunk 1]
```diff
@@ -74,6 +74,23 @@
         <dependency org="com.fasterxml.jackson.core" name="jackson-databind" rev="2.5.1" /> 
         <dependency org="com.fasterxml.jackson.dataformat" name="jackson-dataformat-cbor" rev="2.5.1" />
         <dependency org="com.fasterxml.jackson.jaxrs" name="jackson-jaxrs-json-provider" rev="2.5.1" />	
+        
+        <!-- WebUI -->
+        <dependency org="org.apache.wicket" name="wicket-core" rev="6.16.0" conf="*->default" />
```
Comment (lewismc) [inline ivy/ivy.xml]: Wicket is up at 7 any reason not to use it?

File: src/bin/nutch [hunk 2]
```diff
@@ -87,6 +87,7 @@ if [ $# = 0 ]; then
   echo "  plugin            load a plugin and run one of its classes main()"
   echo "  junit             runs the given JUnit test"
   echo "  startserver	    runs the Nutch Server on localhost:8081"
+  echo " webapp         run a local Nutch web application"
```
Comment (lewismc) [inline src/bin/nutch]: indentation

File: src/java/org/apache/nutch/webui/model/NutchConfig.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package org.apache.nutch.webui.model;
```
Comment (lewismc) [inline src/java/org/apache/nutch/webui/model/NutchConfig.java]: License header

File: src/java/org/apache/nutch/webui/pages/instances/InstancePanel.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
```
Comment (lewismc) [inline src/java/org/apache/nutch/webui/pages/instances/InstancePanel.html]: License header

File: src/java/org/apache/nutch/webui/pages/instances/InstancePanel.java [hunk 5]
```diff
@@ -0,0 +1,62 @@
+package org.apache.nutch.webui.pages.instances;
```
Comment (lewismc) [inline src/java/org/apache/nutch/webui/pages/instances/InstancePanel.java]: License header

File: src/java/org/apache/nutch/webui/pages/instances/InstancesPage.html [hunk 6]
```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
```
Comment (lewismc) [inline src/java/org/apache/nutch/webui/pages/instances/InstancesPage.html]: License header

File: src/java/org/apache/nutch/webui/pages/instances/InstancesPage.java [hunk 7]
```diff
@@ -0,0 +1,111 @@
+package org.apache.nutch.webui.pages.instances;
```
Comment (lewismc) [inline src/java/org/apache/nutch/webui/pages/instances/InstancesPage.java]: License header

File: src/java/org/apache/nutch/webui/pages/menu/VerticalMenu.java [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+ */
+package org.apache.nutch.webui.pages.menu;
+
+import de.agilecoders.wicket.core.markup.html.bootstrap.navbar.Navbar;
+
+public class VerticalMenu extends Navbar {
+
+  public VerticalMenu(String componentId) {
+    super(componentId);
+    // TODO Auto-generated constructor stub
```
Comment (lewismc) [inline src/java/org/apache/nutch/webui/pages/menu/VerticalMenu.java]: remove

File: src/java/org/apache/nutch/webui/pages/settings/SettingsPage.html [hunk 9]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```
Comment (lewismc) [inline src/java/org/apache/nutch/webui/pages/settings/SettingsPage.html]: License header

File: src/java/org/apache/nutch/webui/pages/settings/SettingsPage.java [hunk 10]
```diff
@@ -0,0 +1,59 @@
+package org.apache.nutch.webui.pages.settings;
```
Comment (lewismc) [inline src/java/org/apache/nutch/webui/pages/settings/SettingsPage.java]: License header

**Comment by chrismattmann** [comment]
@sujen1412 FYI these comments from @lewismc

**Comment by sujen1412** [comment]
@lewismc thanks for your comments, will update the PR.

**Comment by chrismattmann** [comment]
any update here? @sujen1412

**Comment by sujen1412** [comment]
@chrismattmann, closing this one creating a new PR with JIRA issue. And before committing the webui I have to get https://github.com/apache/nutch/pull/59 committed.

**Comment by chrismattmann** [comment]
OK #59 is committed now. So I'll close this one out. Then please open a new one with the integration :) :+1:

**Pull request closed without merging**